guessingGame.js

```diff
@@ -0,0 +1,34 @@
+pry = require('pryjs')
+
+function guess(number){
+  if(number>this.answer){
+    console.log('high');
+    return 'high';
+  }else if (number<this.answer){
+    console.log('low');
+    return 'low'
+  }else{
+    console.log("correct");
```

Comment:
no need to console log information here. when you return the value, that is enough 👍